deploy/base/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default
```

Comment:
Go with better default settings in base. Like grafana-operator-system or grafana